dev/build_release.py

```diff
@@ -222,6 +218,9 @@ def __debian_build(cls, name, options, build_number, gradle_root):
       '-PbintrayUser="{user}"'.format(user=bintray_user)
     ]
 
+    if options.maven_custom_init_file:
+      init_file_path = os.path.abspath(options.maven_custom_init_file)
```

Comment:
why do you need the abspath here?

Reply:
Jenkins's workspace directory symlinking throws off the path building.

Reply:
Is this really still a problem after the adding the path below to the default value?

Reply:
We'll see, I'll try it.

Reply:
Seems like it works.